martian/src/mro.rs

```diff
@@ -653,15 +648,15 @@ impl StructHeader {
             for field in &def.fields {
                 self.add_mro_field(field);
```

Comment:
Adam, the order is implicit here: this function is recursive, and evaluates inner fields before outer ones.